<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Macros</class>
 <widget class="QWidget" name="Macros">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="confirmBtn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>360</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>360</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="keyFuc">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>411</width>
     <height>341</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>鼠标功能</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>按键</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>多按键宏</string>
    </attribute>
    <widget class="QTextEdit" name="keyText">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>221</width>
       <height>311</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="recordHandlerTmr">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>171</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>记录事件之间的延迟</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startRecordBtn">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>开始录制</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
